liota/entities/systems/dk300_system.py

```diff
@@ -0,0 +1,8 @@
+from system import System
+from liota.utilities.utility import systemUUID
+
+
+class Dk300System(System):
+
+    def __init__(self, name):
+        super(Dk300System, self).__init__(name=name, entity_id=systemUUID().get_uuid(name))
```

Comment:
If entity_id is always generated, let us move this call to the base Entity call. Let us not call this from every class we are defining.